### DISCUSSION TITLE
No school-issued email

### DISCUSSION THREAD

**Discussion by INotSleep** [community/community#138181] [GitHub Education]
### Hi everyone,

Hi everyone, 

Im studing in univercity in Ukraine. When i choose my univercity, it says that i need to have school-issued email adress, but as said faculty methodologist, students will not recieve their emails.
What to do?

Also i tried to write name manualy to go to map and choose univercity on map, but it shows as old name. My application was auto-rejected.

**Comment by alx-mp4** [comment]
Hi @INotSleep, if you do not have a school issued email, I am afraid you won't be able to get the benefits.
